frontend/src/libs/components/sidebar-wrapper/sidebar-wrapper.tsx

```diff
@@ -0,0 +1,20 @@
+import home from '#assets/img/home.svg';
+import { Sidebar } from '#libs/components/components.js';
+import { AppRoute } from '#libs/enums/enums.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  children: React.ReactNode;
+};
+
+const SidebarWrapper: React.FC<Properties> = ({ children }) => {
+  return (
+    <div className={styles['app']}>
+      <Sidebar routes={[{ path: AppRoute.ROOT, name: 'home', icon: home }]} />
```

Comment:
Let's move routes to a separate variable, it'll be easier to manage when we will be adding new routes

Reply:
we have the same approach for app routes, so let's keep it

Reply:
Done, if I understood correctly